File: DB-Service/Models/Task.cs
namespace DB_Service.Models
{
    public class Task: BaseEntity
    {
        public int? StageId { get; set; }
        public string Title { get; set; }
        public Stage? Stage { get; set; }
        public DateTime? StartedAt { get; set; }
        public DateTime? EndVerificationDate { get; set; } // добавить endpoint
        public DateTime? ApprovedAt { get; set; } // утверждаем
        public TimeSpan ExpectedTime { get; set; }
        public string? Signed { get; set; }
        public int? SignId { get; set; }
        public ICollection<Comment> Comments { get; set; } = new List<Comment>();
    }
}

File: web/src/shared/interfaces/task.ts
import { Comment } from '../../shared/interfaces/comment'
import { User } from './user'

export interface Task {
	id: number
	title: string
	stageId: number
	signId: number
	startedAt: string
	approvedAt: string
	expectedTime: string
	signed: string
	user: User
	comments: Comment[]
	endVerificationDate: string
}
